microsoft-365/business/access-resources.md

```diff
@@ -38,6 +38,8 @@ Complete the following steps to enable your organization's Azure AD joined devic
 3. Once the Windows 10 devices are Azure AD joined, each user must reboot their devices and sign in with their Microsoft 365 Business credentials. All devices now have access to on-premises resources as well.
     
 No additional steps are required to get access to on-premises resources for Azure AD joined devices. This functionality is built into Windows 10. 
+
+If you have plans to login to the AADJ device other than password method Like PIN/Bio-metric via WHFB credential login and then access on-premise resources (shares,printers..etc), please follow https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base
```

Comment:
```suggestion
If you do not plan to log in to the AADJ device using the password method but would rather log in using another method like PIN/Bio-metric via WHFB credential login and then access on-premise resources (shares, printers, etc.), refer to [Configure Azure AD joined devices for On-premises Single-Sign On using Windows Hello for Business](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base).
```

Reply:
That's sound good to me. Please proceed.